ee/maintained-apps/outputs/microsoft-teams/darwin.json

```diff
@@ -1,21 +1,21 @@
 {
```

Comment:
TODO diff the scripts

Reply:
```diff
=== Install Script (no changes) ===
=== Uninstall // 60c2994d -> 61872d1e ===

--- /var/folders/_v/23d_08t17r799nn2d6rdzp0h0000gn/T/old.2Wh7n5KByc	2025-07-31 18:29:16
+++ /var/folders/_v/23d_08t17r799nn2d6rdzp0h0000gn/T/new.AFnhyOMoiF	2025-07-31 18:29:16
@@ -180,8 +180,6 @@
 quit_application 'com.microsoft.autoupdate2'
 remove_pkg_files 'com.microsoft.MSTeamsAudioDevice'
 forget_pkg 'com.microsoft.MSTeamsAudioDevice'
-remove_pkg_files 'com.microsoft.package.Microsoft_AutoUpdate.app'
-forget_pkg 'com.microsoft.package.Microsoft_AutoUpdate.app'
 remove_pkg_files 'com.microsoft.teams2'
 forget_pkg 'com.microsoft.teams2'
 sudo rm -rf '/Applications/Microsoft Teams.app'
```

Reply:
This is actually an improvement over the previous uninstall script, as we _shouldn't_ be removing MS AutoUpdate. Good to go.